docs/website/layouts/index.redirects

```diff
@@ -4,6 +4,22 @@
 # Redirect to latest doc version by default
 /docs     /docs/latest
 
+# v0.14.0 re-org redirects
+# TODO: Remove at some point in the future when its more ok to break old links en-mass
+/docs/latest/how-does-opa-work/                     /docs/latest/#how-does-opa-work
+/docs/latest/how-do-i-write-policies/               /docs/latest/policy-language/
+/docs/latest/how-do-i-test-policies/                /docs/latest/policy-testing/
+/docs/latest/language-reference/                    /docs/latest/policy-reference/
+/docs/latest/language-cheatsheet/                   /docs/latest/edge/policy-cheatsheet/
+/docs/latest/configuration/                         /docs/latest/management/#configuration-reference
+/docs/latest/rest-api/                              /docs/latest/integration/#integrating-with-the-rest-api
```

Comment:
Was this one intentional?

Reply:
Good catch! I was messing around with them and forgot to remove that. Update incoming.